package.json

```diff
@@ -26,7 +25,7 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^5.1.0",
     "eslint-plugin-typescript": "^0.14.0",
-    "fastify": "^3.2.1",
+    "fastify": "^3.24.1",
```

Comment:
Is there a specific reason for this version bump?

Reply:
considering the lack of package-lock, it's a noop anyway, just makes it easier to run ncu.